### DISCUSSION TITLE
Only allow integer numbers in InputNumber

### DISCUSSION THREAD

**Discussion by timonmasberg** [NG-ZORRO/ng-zorro-antd#8497] [General]
Is it possible to only allow positive integers in the InputNumber component? I wasnt able to find any hint in the docs.

**Comment by ParsaArvanehPA** [comment]
Greetings @timonmasberg ,
Sorry for the long delay; have your tried using `nzMin`? 
The [basic section](https://ng.ant.design/components/input-number/en#components-input-number-demo-basic) of the `nzInputNumber` has the same behaviuor you are looking for.

**Reply by timonmasberg** [reply]
Yes, with nzMin i can have positive values only, but it can also be a floating-point value. I am aware, that in Typescript all numbers are floating-point values, so my question was if the component might be able to restrict the input to Integers.
